docroot/config/sync/acquia_connector.settings.yml

```diff
@@ -0,0 +1,210 @@
+cloud_migration: {  }
+subscription_data:
+  active: true
```

Comment:
Please set the subscription to false by default and enable it only on Acquia environments.